results/benchmarks.json

```diff
@@ -1147,6 +1147,94 @@
         "version": "e17b327697d267cf044412cc544be3b2e309edad29b10d0ab5af64a7b847d269",
         "warmup_time": -1
     },
+    "ag_methods.CompoundSplitting.time_center_compounds": {
```

Comment:
Ignore these changes... they should have been on master but I added them to the wrong branch.